release_notes/ocp-4-13-release-notes.adoc

```diff
@@ -99,6 +99,15 @@ With {product-title} {product-version}, running `oc describe` on an image now re
 [id="ocp-4-13-machine-api"]
 === Machine API
 
+[id="ocp-4-13-mapi-cpms-platform-support"]
+==== Additional platform support for control plane machine sets
+
+This release includes an enhancement to the user experience for the control plane machine set on Microsoft Azure clusters.
+
+For Azure clusters that are installed with or upgraded to {product-title} version 4.13, you are no longer required to create a control plane machine set custom resource (CR). Instead, an inactive CR is generated for the cluster and can be activated after you verify that the values in the CR are correct for your control plane machines.
```

Comment:
@jeana-redhat Not sure if I understand correctly, but for fresh install, CPMS is in Active state. If delete or upgrade to 4.13, CPMS is InActive.

Reply:
Ah yes. That is true. Sorry for confusion @jeana-redhat.

Reply:
Ok, so I think that means that in #55407, the table should actually look very similar for AWS and Azure (just different versions). I will fix this PR and update the other to match, then tag for validation again :)

Reply:
Updated this PR and the other one. @odvarkadaniel and @sunzhaohua2 PTAL :)

Reply:
Thanks @jeana-redhat 
LGTM